knexfile.js

```diff
@@ -0,0 +1,48 @@
+require("dotenv").config();
```

Comment:
Huh. Interesting that you put this here! Any particular reason you prefer it here over, say, index.js?

Reply:
I think it's for foolproof in case it doesn't get injected in time, seen it happen with buggy knex versions

Reply:
I'm using environment variables here and it won't pick unless that is there. There's also another one in the server.js